stix_shifter/stix_transmission/src/modules/azure_sentinel/azure_sentinel_api_client.py

```diff
@@ -0,0 +1,59 @@
+from ..utils.RestApiClient import RestApiClient
+
+
+class APIClient:
+    """API Client to handle all calls."""
+    def __init__(self, connection, configuration):
+        """Initialization.
+        :param connection: dict, connection dict
+        :param configuration: dict,config dict"""
+
+        headers = dict()
+        url_modifier_function = None
+        auth = configuration.get('auth')
+        self.endpoint = 'beta/security/alerts'
```

Comment:
just wondering, do you know when they are removing the `beta/ ` from the endpoint path?

Reply:
@mdazam1942 The current code is now modified to take an optional configuration parameter(api_version), which could be either 'beta' or 'v1.0'. By default, if the parameter is not supplied the api_version is set to 'v1.0' for building the sentinel API endpoint.

Reply:
this is better.i replied to your email and asked few question about how the user would get the api version. If the user know the version then we can ask them to put in the connection object.

I would actually put that inside `connection` object in the options attribute. like this:

`api_version = connection.get('options', {}).get('api_version', default_api_version)`

Reply:
The api-version information for Azure sentinel alerts endpoint is not available in the console, we can obtain that from Microsoft public technical documents only. As preferred, we have reverted the connector code to utilize the hard-coded api- version(‘v1.0’).

Reply:
sounds good